### ISSUE TITLE
Vagrant attempted to execute the capability 'os_variant'

### ISSUE THREAD

**Issue by errordeveloper** [projectatomic/vagrant-service-manager#220]
I've installed this plugin to use CDKv2, but it appears to break other things for me:

```
> cat Vagrantfile 
Vagrant.configure(2) do |config|
  config.vm.box = "debian/jessie64"
end

> vagrant version
Installed Version: 1.8.1
Latest Version: 1.8.1

You're running an up-to-date version of Vagrant!

> vagrant plugin list
vagrant-aws (0.7.0)
vagrant-awsinfo (0.0.8)
vagrant-hostmanager (1.8.1)
vagrant-registration (1.2.1)
vagrant-service-manager (1.0.1)
vagrant-share (1.1.5, system)
vagrant-vmware-appcatalyst (1.1.1)

> vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'debian/jessie64'...
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'debian/jessie64' is up to date...
==> default: Setting the name of the VM: weave-net-docker-and-rkt_default_1463559003209_36239
==> default: Clearing any previously set network interfaces...
==> default: When using virtualbox, a non-NAT network interface is required.
==> default: Adding a private network using DHCP
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: hostonly
==> default: Forwarding ports...
    default: 22 (guest) => 2222 (host) (adapter 1)
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: 
    default: Vagrant insecure key detected. Vagrant will automatically replace
    default: this with a newly generated keypair for better security.
    default: 
    default: Inserting generated public key within guest...
    default: Removing insecure key from the guest if it's present...
    default: Key inserted! Disconnecting and reconnecting using new SSH key...
==> default: Machine booted and ready!
==> default: Checking for guest additions in VM...
    default: No guest additions were detected on the base box for this VM! Guest
    default: additions are required for forwarded ports, shared folders, host only
    default: networking, and more. If SSH fails on this machine, please install
    default: the guest additions and repackage the box to continue.
    default: 
    default: This is not an error message; everything may continue to work properly,
    default: in which case you may ignore this message.
==> default: Configuring and enabling network interfaces...
Vagrant attempted to execute the capability 'os_variant'
on the detect guest OS 'debian', but the guest doesn't
support that capability. This capability is required for your
configuration of Vagrant. Please either reconfigure Vagrant to
avoid this capability or fix the issue by creating the capability.
```

It works as expected when I remove the plugin:

```
> vagrant destroy -f
==> default: Forcing shutdown of VM...
==> default: Destroying VM and associated drives...

> vagrant plugin uninstall vagrant-service-manager
Uninstalling the 'vagrant-service-manager' plugin...

> vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'debian/jessie64'...
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'debian/jessie64' is up to date...
==> default: Setting the name of the VM: weave-net-docker-and-rkt_default_1463559071232_59094
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
==> default: Forwarding ports...
    default: 22 (guest) => 2222 (host) (adapter 1)
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: 
    default: Vagrant insecure key detected. Vagrant will automatically replace
    default: this with a newly generated keypair for better security.
    default: 
    default: Inserting generated public key within guest...
    default: Removing insecure key from the guest if it's present...
    default: Key inserted! Disconnecting and reconnecting using new SSH key...
==> default: Machine booted and ready!
==> default: Checking for guest additions in VM...
    default: No guest additions were detected on the base box for this VM! Guest
    default: additions are required for forwarded ports, shared folders, host only
    default: networking, and more. If SSH fails on this machine, please install
    default: the guest additions and repackage the box to continue.
    default: 
    default: This is not an error message; everything may continue to work properly,
    default: in which case you may ignore this message.
==> default: Installing rsync to the VM...
==> default: Rsyncing folder: /Users/ilya/Code/weave-net-docker-and-rkt/ => /vagrant
```

**Comment by brgnepal** [comment]
Thanks @errordeveloper for reporting it.

Fix  already sent [here](https://github.com/projectatomic/vagrant-service-manager/pull/221) and available in next released version `1.0.3`.

**Comment by rnaszcyn** [comment]
Looking forward to being able to install 1.0.3. Without fix I must uninstall plugin to use other installs that rely on vagrant.
